### PR TITLE
created the login package inside interface_adapter, created LoginCont…

### PR DESCRIPTION
**Pull request by sedakk**
…roller, LoginPresenter, LoginState,LoginViewModel, and imported LoginState and LoginViewModel into SignupPresenter

### CODE REVIEW

**Comment by AasthaSharma272** [comment]
Hi, 
I do see a conflict in SignupPresenter.java, so not merging it in right now. Will look into the conflict and see what changed.

File: src/interface_adapter/SignupPresenter.java
Comment (AasthaSharma272) [inline src/interface_adapter/SignupPresenter.java:1]: Hi, 
I think the conflict arises on this file as you got rid of the import of ViewManagerModel. Can you add that back as this file requires that import. Will merge it when it is added.

**Review by AasthaSharma272 — CHANGES_REQUESTED** [review]
Hi, on the src/interface_adapter/SignupPresenter.java can you please add the import of ViewManagerModel back as that is causing a conflict. When you add it, we can merge the files.

File: src/interface_adapter/SignupPresenter.java
Comment (sedakk) [reply src/interface_adapter/SignupPresenter.java:1]: Hey, sorry about that! Just added it back in the newest commit

**Review by AasthaSharma272 — APPROVED** [review]
Looks good!

**Pull request merged**